ModelFactoryCollection/ModelFactoryCollection.php

```diff
@@ -73,10 +73,17 @@ public function createModels($objects)
         foreach ($objects as $key => $object) {
             $modelFactory = $this->findSupportingModelFactoryForObject($object);
             if (!$modelFactory instanceof ModelFactoryInterface) {
-                throw new ModelFactoryCollectionException(sprintf(
-                    "Model factory for class '%s' not found.",
-                    get_class($object)
-                ));
+                if (is_object($object)) {
+                    throw new ModelFactoryCollectionException(sprintf(
+                        'Model factory for class %s not found.',
+                        get_class($object)
+                    ));
+                } else {
```

Comment:
Yeah, the `else` is obsolete, because there is a `throw` statement above. Thanks!